ramls/oaipmh/instanceids.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
Both RTAC and OAI-PMH requst the same API in mod-inventory-storage. I think the confusion was introduced while copypasting.  I have doubts too regarding their usage since this is not an RMB-based module. I guess, I'll remove them

Reply:
done